src/tests/test_sexual_behaviour.py

```diff
@@ -104,3 +106,225 @@ def test_initial_sex_behaviour_groups():
             Expectation = float(n_sex)*p
             N_g = sum(index_sex & index_group)
             assert Expectation - sigma*3 <= N_g <= Expectation + sigma*3
+
+
+def test_rred_long_term_partner():
+    N = 1000
+    pop = Population(size=N, start_date=date(1989, 1, 1))
+    # pick some indices and give those people LTPs
+    indices = [randint(0, N-1) for i in range(15)]
```

Comment:
We could use the numpy generator for more consistency:
```suggestion
    indices = rng.integers(0, N, size=15)
```
And similar in some other test functions. Confusingly, the numpy functions are exclusive of the high argument, unlike the standard library's.

(it wouldn't really change anything functionally, but as a general guide I suggest we avoid the standard library `random`, so we don't get confused about which seed has been set)